apps/opencs/view/render/terrainstorage.hpp

```diff
@@ -1,27 +1,49 @@
 #ifndef OPENCS_RENDER_TERRAINSTORAGE_H
 #define OPENCS_RENDER_TERRAINSTORAGE_H
 
+#include <array>
+
 #include <components/esmterrain/storage.hpp>
 
 #include "../../model/world/data.hpp"
 
 namespace CSVRender
 {
-
     /**
      * @brief A bridge between the terrain component and OpenCS's terrain data storage.
      */
     class TerrainStorage : public ESMTerrain::Storage
     {
     public:
         TerrainStorage(const CSMWorld::Data& data);
+        std::array<float, ESM::Land::LAND_SIZE * ESM::Land::LAND_SIZE> mAlteredHeight;
```

Comment:
Make this field private since it's not used outside of this class and there are public methods to get all required access.